selene_sdk/predict/model_predict.py

```diff
@@ -1143,3 +1137,16 @@ def variant_effect_prediction(self,
 
         for r in reporters:
             r.write_to_file()
+
+    def _pad_or_truncate_sequence(self, input_seq):
+        sequence = input_seq
```

Comment:
remove this line, use directly `_pad_or_truncate_sequence(self, sequence)`. thanks for the refactor, I like this helper function. :)